src/Components/StartStopButton/StartStopButton.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const StartStopButton = ({ startHandler, stopHandler }) => {
+  return (
+    <>
+      <button onClick={startHandler}>Start</button>
+      <button onClick={stopHandler}>Stop</button>
+    </>
+  );
+};
+
+export default StartStopButton;
```

Comment:
I think that it's better to put the buttons directly inside the countDown.js file, and delete this file.